gateway/assets/src/page-header.html

```diff
@@ -0,0 +1,14 @@
+<div id="page-header">
+  <div id="page-header-logo" class="ipfs-logo">&nbsp;</div>
+  <div id="page-header-menu">
+    <div class="menu-item-wide"><a href="https://ipfs.tech" target="_blank" rel="noopener noreferrer">About IPFS</a></div>
+    <div class="menu-item-wide"><a href="https://ipfs.tech#install" target="_blank" rel="noopener noreferrer">Install IPFS</a></div>
+    <div class="menu-item-narrow"><a href="https://ipfs.tech" target="_blank" rel="noopener noreferrer">About</a></div>
+    <div class="menu-item-narrow"><a href="https://ipfs.tech#install" target="_blank" rel="noopener noreferrer">Install</a></div>
+    <div>
+      <a href="https://github.com/ipfs/kubo/issues/new/choose" target="_blank" rel="noopener noreferrer" title="Report a bug">
```

Comment:
I would remove the kubo issue opening from here.  It's not high signal that a Kubo issue should be opened (especially since this is in Boxo).

Reply:
+1 for removing this from default template. 

As part of customization, we could add it as opt-in via config (`Gateway.SupportURL`) and set it to on ipfs.io and dweb.link  to boxo repo with some gateway-specific template, but we should not expose it on third-party gateways, which may be misconfigured.

Reply:
This is part of the common header between all pages. If "but we should not expose it on third-party gateways" is the reasoning, we should remove it from **everywhere** (directories, dag listings, errors). Is that the case?

Reply:
I removed Kubo's URL and replaced it by a customisable `.SupportURL`. Whoever implements the gateway can now pass that information. If it is set, it will show up with the custom URL. If not, it simply doesn't show up.